README.md

```diff
@@ -1 +1,176 @@
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+- Download a text editor. example Visual Studio Code
+
+### Setup
+
+Clone this repository to your desired folder:
+
+  Clone this repository to your desired folder
+  - git clone https://github.com/Blessman-Newton/Microverse_Porfolio.git
+
+### Install
+
+Install this project with:
+
+- Install this project with: The project is installed once you clone it on your local machine
+
+### Usage
+
+To run the project, execute the following command:
+
+Open index.html in your browser
+
+### Run tests
+
+To run tests, run the following command:
+
+<npx stylelint "**/*.{css,scss}
+
+### Deployment
+
+You can deploy this project using:
+
+You can deploy this project using by transfering files in your local repository onto your server.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 👥 Authors <a name="authors">Blessman Newton</a>
+
+
+👤 **Author1**
+
+- GitHub: [@Blessman Newton](https://github.com/Blessman-Newton)
+- Twitter: [@Blessman Newton](https://twitter.com/blessman_newton)
+- LinkedIn: [Blessman Newton](https://linkedin.com/in/blessman-newton-a7a80a251)
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🔭 Future Features <a name="future-features"></a>
+
+
+- [ ] **[Section to show my resume]**
+- [ ] **[Section to show languages that I have learned]**
+- [ ] **[Contact form to contact me]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project, show me 🤝🤝🤝
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank Microverse for the best technical training they are taking me through and the code reviewers for having guided me to produce required results
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## ❓ FAQ (OPTIONAL) <a name="faq"></a>
+
+- **[Question_1]**
+
+  - [Answer_1]
+
+- **[Question_2]**
+
+  - [Answer_2]
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
+
+_NOTE: we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish._
+
```

Comment:
- [OPTIONAL] I'll suggest you remove this as this is just a placeholder comment provided by Microverse to help you setup your license file. :waffle: